lib/widgets/action_sheet.dart

```diff
@@ -163,6 +163,59 @@ class ActionSheetCancelButton extends StatelessWidget {
   }
 }
 
+/// Show a sheet of actions you can take on a channel.
+void showChannelActionSheet(BuildContext context, {
+  required int channelId,
+}) {
+  final pageContext = PageRoot.contextOf(context);
+  final store = PerAccountStoreWidget.of(pageContext);
+
+  final optionButtons = <ActionSheetMenuItemButton>[];
+  final unreadCount = store.unreads.countInChannelNarrow(channelId);
+  if (unreadCount > 0) {
+    optionButtons.add(
+      MarkChannelAsReadButton(
+        streamId: channelId,
+        pageContext: pageContext,
+      ),
+    );
```

Comment:
```suggestion
    optionButtons.add(
      MarkChannelAsReadButton(pageContext: pageContext, streamId: channelId));
```

(context first, and fewer lines of code)